pandas/_libs/lib.pyx

```diff
@@ -199,7 +199,7 @@ def item_from_zerodim(val: object) -> object:
 
 @cython.wraparound(False)
 @cython.boundscheck(False)
-def fast_unique_multiple(list arrays):
+def fast_unique_multiple(list arrays, sort: bool=True):
```

Comment:
``MultiIndex.union`` calls this function so I added a ``sort`` parameter to control the behaviour here and set the default to ``True`` for compatibility

Reply:
can you add a doc-string here